### PR TITLE
Write to error stream only if exit code is non-zero

### PR DESCRIPTION
**Pull request by ds-ms**
Fixes https://github.com/microsoft/azure-pipelines-tasks/issues/10575

### CODE REVIEW

File: Tasks/KubernetesManifestV0/src/utils/utilities.ts [hunk 1]
```diff
@@ -59,8 +59,10 @@ export function checkForErrors(execResults: IExecSyncResult[], warnIfError?: boo
     if (execResults.length !== 0) {
         let stderr = '';
         execResults.forEach(result => {
-            if (result.stderr) {
+            if (result.code !== 0 && result.stderr) {
                 stderr += result.stderr + '\n';
+            } else if (result.stderr) {
```
Comment (nishubansal) [inline Tasks/KubernetesManifestV0/src/utils/utilities.ts]: Redundant check for if (result.stderr). You can change it to something like 
if (result.stderr) { 
                if (result.code !== 0) {stderr += result.stderr + '\n';} else { tl.warning(result.stderr);}

File: Tasks/KubernetesManifestV0/task.loc.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 0,
     "Minor": 154,
```
Comment (vithati) [inline Tasks/KubernetesManifestV0/task.loc.json]: 0.155.0

Comment (ds-ms) [reply Tasks/KubernetesManifestV0/task.loc.json]: There are multiple fixes going in. I'll update the last one with `0.155.0`

**Comment by ds-ms** [comment]
Closed and reopened to trigger tests

**Pull request merged**